### ISSUE TITLE
[FR] emoji and icons changer

### ISSUE THREAD

**Issue by HitaloSama** [NekoX-Dev/NekoX#333]
Add the option to change the font of emojis, it is present in Nekogram but in Nekogram X no, it would also be good to change the Telegram icons between the originals and those of Telegram X...

**Comment by lluucc** [comment]
I agree

**Comment by feedelphix** [comment]
Agreed too. By the way Nekogram X needs icons that have similar look and not pixelated and so it might solve this problem.

**Comment by microtechton** [comment]
To take this one step further, it would be nice if there was a custom option and we could load our own font file. 

Speaking of changing the emoji font, it would be nice to load a custom ttf font to NekoX

**Comment by HitaloSama** [comment]
> To take this one step further, it would be nice if there was a custom option and we could load our own font file. 
> 
> Speaking of changing the emoji font, it would be nice to load a custom ttf font to NekoX 

Yes, in normal Nekogram it is possible to use any ttf of compatible font, Nekogram X uses the twemojis, which in my view are very ugly, give the option of choice to the user would be great.
